modules/build/src/main/scala/scala/build/Artifacts.scala

```diff
@@ -129,12 +129,23 @@ object Artifacts {
         Nil
     }
 
+    val scala2NightlyRepo = Seq(coursier.Repositories.scalaIntegration.root)
+
     val scalaNativeCliDependency =
       scalaNativeCliVersion.map(version =>
         Seq(dep"org.scala-native:scala-native-cli_2.12:$version")
       )
 
-    val allExtraRepositories = maybeSnapshotRepo ++ extraRepositories
+    val scala2NightlyRegex = raw"""(\d+)\.(\d+)\.(\d+)-bin-[a-f0-9]*""".r
```

Comment:
I think, that this regex should start from `2`

Reply:
What do you mean?

Reply:
I think, this regex should works for input `3.13.2-bin-c4434f`.